.github/workflows/unroute-new-issue.yml

```diff
@@ -0,0 +1,33 @@
+name: Unroute new issue
```

Comment:
"Unroute" sounds like you are undoing something, how about this
```suggestion
name: Mark new issue for routing
```

Reply:
https://github.com/getsentry/sentry/pull/26964#issuecomment-870913818